.github/workflows/update-schema.yml

```diff
@@ -0,0 +1,93 @@
+name: Update schema
+
+concurrency: update-schema-${{ github.ref }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "New okp4/contracts version (repository tag or repository ref)"
+        required: true
+      draft:
+        type: boolean
+        description: "Indicate if schemas need to be only publish in draft or need to be released and published publicly"
+        required: false
+        default: false
+
+jobs:
+  update-schema:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.OKP4_TOKEN }}
+          fetch-depth: 2
+
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.version }}
```

Comment:
The checkout is already restricted to `okp4-contracts` since its configured on the `repository` params of `action/checkout@v3` parameters, I could add check if the ref given is the `main` branch (for draft) or `v*` for release.

Reply:
Ah, alright. Seems good then. :+1: